android/expoview/build.gradle

```diff
@@ -41,6 +42,16 @@ def safeExtGet(prop, fallback) {
   rootProject.ext.has(prop) ? rootProject.ext.get(prop) : fallback
 }
 
+def getLastSupportedABI(){
+  def lastSupportedVersion = "abi_999_0_0"
+  (gradle.ext.has("supportedABIS") ? gradle.ext.supportedABIS : ["abi_0_0_0"]).forEach ({ version ->
+    if (version < lastSupportedVersion) {
```

Comment:
you sure this will work? 😜

Reply:
I've tested it and everything was working fine.
But I'm not sure if this behaviour is what we want. 
I assume that if `gradle.ext.supportedABIS` doesn't exist then annotation shouldn't throw errors (that's why I've passed `abi_0_0_0 ` instead of `supportedABIS`).